<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPrincipal</class>
 <widget class="QDialog" name="TelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de Registros</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1201</width>
     <height>721</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Funcion√°rios</string>
    </attribute>
    <widget class="QPushButton" name="btnCadastrarFuncionario">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>   Novo</string>
     </property>
     <property name="icon">
      <iconset>
       <selectedon>:/Imagens/Imagens/novo.png</selectedon>
      </iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPesquisarFuncionario">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>160</y>
       <width>631</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>130</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Digite algo para pesquisar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExcluirFuncionario">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>    Excluir</string>
     </property>
     <property name="icon">
      <iconset resource="recursosimagens.qrc">
       <normaloff>:/Imagens/Imagens/excluir.png</normaloff>:/Imagens/Imagens/excluir.png</iconset>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetFuncionario">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>1111</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalRegistros">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>610</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total de Registros</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Filtro</string>
     </property>
     <widget class="QRadioButton" name="rdCPF">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdNome">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdDepartamento">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Departamento</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Clientes</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recursosimagens.qrc"/>
 </resources>
 <connections/>
</ui>
